scripts/repair_wheel.py

```diff
@@ -0,0 +1,91 @@
+import argparse
```

Comment:
```suggestion
#!/usr/bin/env python3

import argparse
```

And it could be executable? You require at least 3.5 (from  Pathlib and exist_ok), so at the very least this tells a reader this is a python3 script.

Reply:
I'm doubtful about the usefulness of making this one executable but just to be consistent with the shebang which suggests it is, let's do it.